packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/BasicSecurity/TwoStepVerification/template.success.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Both of these `eslint-disable` rules were added as there are some html anchor elements previous to my work that were causing some conflicts to push my changes.